src/bin/pgcopydb/schema.c

```diff
@@ -3296,7 +3296,6 @@ schema_list_partitions(PGSQL *pgsql,
 	if (splitByCTID)
 	{
 		min = 0;
-		max = table->relpages;
```

Comment:
Can you help me understand this change? I am lacking time to inquire and it probably doesn't even need a comment in the code.

Reply:
The value is assigned but not used.